### ISSUE TITLE
Issue with movecraft

### ISSUE THREAD

**Issue by Cewkaurai** [msummers123/Movecraft-3#263]
https://pastebin.com/gSS4D3d4

That's the pastebin for the latest log of my server; for some reason I can make airskiffs with no issues at all, but when I try and do /pilot and type in any variant of "airship" as part of the name, i.e airship/bigairship, it doesn't say anything in either the log or chat; it just acts like I never ran the command.

The only thing I've changed in the .craft files and the config of the plugin overall is the minimum and maximum blocks for the BigAirship and Airship types. Any ideas as to what the heck is going on here? Any feedback would be appreciated :P 

Oh, and I'm running Spigot on minecraft 1.12, too. I should probably turn on compat mode though... But the wierd thing is, the airskiff type is completely unaffected.

**Comment by hettingerj** [comment]
Can you post your modified craft files?

You have 3 big errors about this:

> [05:15:59] [Server thread/ERROR]: [Movecraft] ***ERROR PARSING CRAFT FILE FROM DIRECTORY : /disk2/multicraft/servers/server41969/plugins/Movecraft/types/BigAirship.craft***

> [05:15:59] [Server thread/ERROR]: [Movecraft] ***ERROR PARSING CRAFT FILE FROM DIRECTORY : /disk2/multicraft/servers/server41969/plugins/Movecraft/types/airskiff.craft***

> [05:15:59] [Server thread/ERROR]: [Movecraft] ***ERROR PARSING CRAFT FILE FROM DIRECTORY : /disk2/multicraft/servers/server41969/plugins/Movecraft/types/airship.craft***`

**Comment by pistols12** [comment]
Here is a good resource to check parsing, it's not 100% bullet proof, but has proven a good tool:

http://yaml-online-parser.appspot.com/

Firstly, I would double check parsing, Secondly, I would let us know when you last updated Movecraft, and if this issue arose AFTER updating, or if these are just new files that have never been working.

If these files were working fine, then when you updated the jar, all the sudden started getting wonkey, I would revert to an older jar.  My personal server was not able to use the latest jar from movecraft, it was causing my server to throw "parsing" errors just like this with KNOWN good files that have been in use for about a year without a hitch. I was forced to revert to my previous movecraft jar.


In short:

A: run all those files through the parser link I gave you

B: try a different jar if you know those files work

c: contact me on skype: Tango1one
